doc/tutorial/read_mapper/read_mapper_step2.cpp

```diff
@@ -45,16 +45,16 @@ void map_reads(std::filesystem::path const & query_path,
         iarchive(index);
     }
 
-    seqan3::sequence_file_input query_in{query_path};
+    seqan3::sequence_file_input query_file_in{query_path};
 
     seqan3::configuration const search_config = seqan3::search_cfg::max_error{seqan3::search_cfg::total{errors}} |
                                                 seqan3::search_cfg::hit_all_best;
 
-    for (auto & [query, id, qual] : query_in | seqan3::views::take(20))
+    for (auto & record : query_file_in | seqan3::views::take(20))
     {
-        auto positions = search(query, index, search_config);
-        seqan3::debug_stream << "id:           " << id << '\n';
-        seqan3::debug_stream << "positions:    " << positions << '\n';
+        seqan3::debug_stream << "Hits:" << '\n';
+        for (auto && res : search(seqan3::get<seqan3::field::seq>(record), index, search_config))
+            seqan3::debug_stream << res << '\n';
```

Comment:
> The result is debug streamable? Do we want this? 

We said in generel we want every public API to be debug streamable. And I wanted this because almost all of the snippets and tutprials print the result of the search and I would have to adapt all of those lines manually, checking which things are available and mentioned in the text. This way was less work.

> Especially, if we say that certain information is only available if they are configured and otherwise throw.

I explicitily check whether the information is present or not and only print the information that's present.